src/Plugin/AbstractIframeValidator.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Drupal\stanford_media\Plugin;
+
+use Drupal\stanford_media\Plugin\EmbedValidatorBase;
+
+/**
+ * Abstract IFrame Validator.
+ *
+ */
+abstract class AbstractIframeValidator extends EmbedValidatorBase {
+
+  const EMBEDDOMAIN = '';
+
+  /**
+   * {@inheritDoc}
+   */
+  public function isEmbedCodeAllowed(string $code): bool {
+    $code = str_replace("\n", ' ', $code);
+    preg_match('/<iframe.* src="(.+?)"/', $code, $source_matches);
+    preg_match('/title\s?=\s?"(.*)"/', $code, $title_matches);
+    if (empty($source_matches[1]) || empty($title_matches[1])) {
+      return FALSE;
+    }
+    $source = parse_url($source_matches[1]);
+    return $source['host'] == self::EMBEDDOMAIN;
```

Comment:
Let's also check for subdomains like the one for google.
```suggestion
    return $source['host'] == self::EMBEDDOMAIN || strpos($source['host'], self::EMBEDDOMAIN) !== FALSE;
```

Reply:
Shouldn't that be an `&&` rather than an OR?

Reply:
i mean, it could just be the strpos condition only. because it will return true if `airtable.com` contains `airtable.com`